lib/tokens/session_token.js

```diff
@@ -22,6 +22,7 @@ module.exports = function (log, inherits, Token) {
     this.verifierSetAt = details.verifierSetAt
     this.locale = details.locale || null
     this.accountCreatedAt = details.createdAt
+    this.tokenVerificationId = details.tokenVerificationId || null
```

Comment:
ditto my earlier questions about whether any verification stuff needs to be included in this PR